client/components/social-buttons/facebook.js

```diff
@@ -0,0 +1,92 @@
+/**
+ * External dependencies
+ */
+import React, { Component, PropTypes } from 'react';
+import SocialLogo from 'social-logos';
+import { loadScript } from 'lib/load-script';
+
+export default class FacebookLoginButton extends Component {
+
+	// See: https://developers.facebook.com/docs/javascript/reference/FB.init/v2.8
+	static propTypes = {
+		appId: PropTypes.string.isRequired,
+		version: PropTypes.string,
+		cookie: PropTypes.bool,
+		status: PropTypes.bool,
+		xfbml: PropTypes.bool,
+		responseHandler: PropTypes.func.isRequired,
+		scope: PropTypes.string,
+	};
+
+	static defaultProps = {
+		version: 'v2.8',
+		cookie: false,
+		status: false,
+		xfbml: true,
+		scope: 'public_profile,email',
+	};
+
+	constructor( props ) {
+		super( props );
+		this.FB = null;
```

Comment:
Super minor, but why use `this.FB` at all? The script populates `window.FB`, which we already reference, so it seems like it muddies the water a bit to access the same object twice through two different references.